vignettes/montagu_user_guide.Rmd

```diff
@@ -440,7 +440,7 @@ the full coverage data for all countries. And for some diseases, the historic ef
 in a country that is not required for estimates as present, may have relevance in their model to
 countries that are required.
 
-For those cases, where all countries are required, the `all_countries` optional argument can be used:
+For those cases, where all countries are required, the following optional argument can be used:
```

Comment:
I feel like "following" is less clear here

Reply:
Agreed - reverted/improved.